CRM/Utils/Mail/FilteredPearMailer.php

```diff
@@ -109,4 +109,8 @@ public function __unset($name) {
     unset($this->_delegate->{$name});
   }
 
+  public function disconnect() {
+    return $this->_delegate->disconnect();
```

Comment:
TBH, I'm a little confused by the fact that `CRM/Mailing/BAO/MailingJob.php` and `src/Listener/DefaultSender.php` both make an unguarded assumption that `disconnect()` is valid method -- since it's only valid with SMTP, and other backends can be configured.

I'd have a soft lean toward guarding this better:

```suggestion
    if (is_callable([$this->_delegate, 'disconnect'])) {
      return $this->_delegate->disconnect();
    }
```